### ISSUE TITLE
In Wordpress 5.9 accessibility issue is coming in block editor panel color setting

### ISSUE THREAD

**Issue by theminaldiwan** [WordPress/gutenberg#39744]
### Description

In Wordpress 5.9 accessibility issue is coming in block editor panel color setting when we open the color palette same time setting panel highlighted "Text" with foreground and background color.
we faced issue is the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.
Elements must have sufficient color contrast.

To solve this issue we need to check element has insufficient color contrast of 4.01(foreground color: #007cba, background color: #f0f0f0, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

### Step-by-step reproduction instructions

1.Go to editor side
2.Open block editor color panel setting
3.Scan the selected portion of color panel setting

### Screenshots, screen recording, code snippet

<img width="826" alt="accesibility-text" src="https://user-images.githubusercontent.com/38693713/160064088-df2c8660-470c-4f10-b25f-1f1aa1947a17.png">
<img width="826" alt="highlight-text" src="https://user-images.githubusercontent.com/38693713/160064098-648798ca-80a3-40ae-bcf4-d7de00805bf6.png">

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by himali-patel** [comment]
Hi Team,
Do we have any specific selected color code which needs to be used?. Or we are good to go for the fix on this ticket?

**Comment by theminaldiwan** [comment]
Hi 
We need to change color contrast ratio from 4.0.1 to above 4.5 .
Thanks

**Comment by Thelmachido** [comment]
> To solve this issue we need to check element has insufficient color contrast of 4.01(foreground color: #007cba, background color: #f0f0f0, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

<img width="584" alt="Untitled" src="https://user-images.githubusercontent.com/41231764/192749815-bf99374d-bd41-47c6-b342-3eb60e0c146d.png">


I wasn't able to see the actual contrast now with the suggested colors, this is what I see. However, you can see that the editor picks up that this is not a good color contrast. I will be closing this issue now. 

WordPress 6.0.1 
Gutenberg 14.1.1

Feel free to re-open if that is not the case
